### DISCUSSION TITLE
API ERROR: Invalid URL (POST /v1/chat/completions)

### DISCUSSION THREAD

**Discussion by drazde** [jackMort/ChatGPT.nvim#92] [Q&A] [answered]
When I try to ask a question, I have this error: `API ERROR: Invalid URL (POST /v1/chat/completions)`

I set `OPENAI_API_KEY` enviroment variable with the relative key.

SO: Windows 11
NVIM v0.9.0-dev-514+g72a19b2ff

**Comment by OdrJunior** [comment, marked as answer]
Hello friend, what model are you using, considering that this endpoint is only available for the gpt-3.5-turbo and higher versions, Here's the link for validation: https://platform.openai.com/docs/models/model-endpoint-compatibility.

**Comment by drazde** [comment]
Ok thanks, 
I had these configurations wrong
```
  openai_params = {
    model = "text-davinci-003",
    frequency_penalty = 0,
    presence_penalty = 0,
    max_tokens = 300,
    temperature = 0,
    top_p = 1,
    n = 1,
  },
  openai_edit_params = {
    model = "code-davinci-edit-001",
    temperature = 0,
    top_p = 1,
    n = 1,
  },
```
